### ISSUE TITLE
Dynamically set animationEnabled config option

### ISSUE THREAD

**Issue by joncursi** [react-navigation/react-navigation#1586]
Consider the following example:

```js
const Tabs = TabNavigator({
   TAB_ONE: { screen: TabOne },
   TAB_TWO: { screen: TabTwo },
}, {
  animationEnabled: false,
});
```

I have disabled animation between the tabs, because when the user taps on a tab, I want the content to display instantly.

However, this doesn't mean I want animation disabled in *all* cases. For example, if a user taps on some touchable target __within__ TabOne, I want that one to animate to TabTwo rather than flash instantly. Different targets have different animation requirements.

Is it possible to programmatically set whether or not tabs should animate? For example, configuring the TabNavigator to default to `false` as in the above example, and then over-riding it on a case-by-case basis. Something along the lines of:

```js
navigation.navigate(TAB_TWO, {
  animationEnabled: true,
});
```

**Comment by lionel95200x** [comment]
I have the same probleme, have you found a solution ?

**Comment by kelset** [comment]
Possible fix #2308 ?

**Comment by brentvatne** [comment]
please re-post to https://react-navigation.canny.io/feature-requests if this is important for you. re-organizing issues to focus on bug reports here and feature requests on canny. thanks!
